File: Java/work9/src/human/Human.java
package human;

class Human {
	String name;
	int age;
	int height;
	int weight;

	Human() {
	}

	Human(String name, int age, int height, int weight) {
		this.name = name;
		this.age = age;
		this.height = height;
		this.weight = weight;
	}

	public String toString() {
		String data = name + "\t" + age + "\t" + height + "\t" + weight;
		return data;
	}
}

File: kotlin/test2/src/main/kotlin/dto/Human.kt
package dto

open class Human {
    var number:Int = 0
    var name:String = ""
    var age:Int = 0
    var height:Double = 0.0


    constructor(){}
    constructor(number: Int, name: String, age: Int, height: Double) {
        this.number = number
        this.name = name
        this.age = age
        this.height = height
    }

    override fun toString(): String {
        return "$number-$name-$age-$height-"
    }
}